d/directory-ldap-api/directory-ldap-api_ubuntu_20.04.sh

```diff
@@ -0,0 +1,52 @@
+# ----------------------------------------------------------------------------
+#
+# Package           : directory-ldap-api
+# Version           : 0.1.0
+# Source repo       : https://github.com/apache/directory-ldap-api
+# Tested on         : ubuntu_20.04
+# Script License    : Apache License, Version 2 or later
+# Maintainer        : Santosh Kulkarni <santoshkulkarni70@gmail.com> / Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer        : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+
+export REPO=https://github.com/apache/directory-ldap-api
+
+if [ -z "$1" ]; then
+  export VERSION="shared-parent-0.9.19"
```

Comment:
Is it 0.1.0 or 0.9.19 ? Also confirm that it is able to build api-asn1-api-1.0.0-M20.jar